src/pages/debrief/cat-cpc/components/debrief-cpc-card/debrief-cpc-card.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Question, Question5 } from '@dvsa/mes-test-schema/categories/CPC';
+
+export type QuestionUnion = Question
+| Question5;
+
+@Component({
+  selector: 'debrief-cpc-card',
+  templateUrl: 'debrief-cpc-card.html',
+})
+export class DebriefCPCCardComponent implements OnInit {
+
+  @Input()
+  public question1: Question;
+
+  @Input()
+  public question2: Question;
+
+  @Input()
+  public question3: Question;
+
+  @Input()
+  public question4: Question;
+
+  @Input()
+  public question5: Question5;
+
+  @Input()
+  public overallScore: number;
+
+  @Input()
+  public questions: QuestionUnion[];
+
+  constructor() {
+  }
+
+  getFormattedQuestion = (question: Question | Question5): string => {
+    return `${question.questionCode} - ${question.title}`;
+  }
+
+  getQuestionPercentage = (question: Question | Question5): number => {
+    return question.score;
+  }
+
+  shouldTickBox = (question: Question | Question5): boolean => {
+    return question.score > 15;
+  }
+
+  getTotalScore = (): number => {
+    let count = 0;
+    for (const question of this.questions) {
+      count = count + this.getQuestionPercentage(question);
+    }
+    return count;
+  }
+
+  isPass = (): boolean => {
```

Comment:
id imagine the simplest way is to add another ()Input which is testOutcome, and grab the value from the store so no need to do any additional calculations in component then

Reply:
yes definitely do not replay the test result calc here. this is already set in the store as Matt says